### DISCUSSION TITLE
clone和改了_congig.yml的theme部分结果报错了。。。求大佬们救救

### DISCUSSION THREAD

**Discussion by Dxin02** [YunYouJun/hexo-theme-starry#293] [Q&A 疑难杂症] [answered]
<img width="446" alt="微信截图_20220513231705" src="https://user-images.githubusercontent.com/87813631/168314758-d39f97e9-dc55-4c16-9eaf-9ff8f576c83d.png">

**Comment by YunYouJun** [comment, marked as answer]
v1.8 之后版本迁移为 npm 安装。

[迁移指南](https://yun.yunyoujun.cn/guide/migrate.html)

**Reply by Dxin02** [reply]
谢谢你，配置好了！
